README.md

```diff
@@ -386,6 +389,51 @@ Ocean Market also includes a number of advanced features that are suitable for a
 
 [See our seperate guide on advanced features](docs/advancedSettings.md)
 
+## ✅ GDPR Compliance
+
+Ocean Market comes with prebuilt components for you to customize to cover GDPR requirements. Find additional information on how to use them below.
+
+### Multi-Language Privacy Policies
+
+Feel free to adopt our provided privacy policies to your needs. Per default we cover four different languages: English, German, Spanish and French. Please be advised, that you will need to adjust some paragraphs in the policies depending on your market setup (e.g. the use of cookies). You can easily add or remove policies by providing your own markdown files in the `content/pages/privacy` directory. For guidelines on how to format your markdown files refer to our provided policies. The pre-linked content tables for these files are automatically generated.
+
+### Privacy Preference Center
+
+Additionally, Ocean Market provides a privacy preference center for you to use. This feature is disabled per default since we do not use any cookies on our deployment of the market. However, if you need to add some functionality depending on cookies, you can simply enable this feature by changing the value of the `privacyPreferenceCenter` variable in the `app.config.js` config file. This will enable a simple cookie banner stating the use of necessary cookies. The content of this banner can be adjusted within the `content/gdpr.json` file. The `gdpr.json` file contains the minimum required values to configure the preference center. However, if you need more customizability for example to configure optional cookies for your market, you can also add these here. To find out how to configure this, we provide an example within the `content` directory. To quickly preview the example you can enable it with the following command:
+
+```
+cd content && mv gdpr.json gdpr_essential.json && cp gdpr.example.json gdpr.json
```

Comment:
It would be nice to add this script to package.json and replace it with something shorter here. For example:

```suggestion
npm run gdpr:preview
```

Reply:
right, I did not get the intent of all the different json files as the app config seems to control showing it or not so that would be enough of a mechanism so we can reduce some complexity here. Would be fine to just have one `content/gdpr.json` in version control and that's it. No renaming and such. Readme makes it sufficiently clear that the app config will then pick up values from there.

Reply:
Yeah, a single gdpr.json file would simplify it

Reply:
I agree it's unnecessary overhead. I removed the example file with one of the latest commits.